### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6437] [Help] [answered]
How to change this icon 

![5](https://user-images.githubusercontent.com/106697208/175142640-5afb2919-7d80-4756-b3dc-64abe66a59db.PNG)

**Comment by melohagan** [comment, marked as answer]
You can provide a logo URL in the layout tab of the **designer**

https://docs.budibase.com/docs/layouts

**Comment by rafikApp** [comment]
Thank you
